src/service/video-manager-impl.js

```diff
@@ -325,6 +364,44 @@ class VideoEntry {
   }
 
   /**
+   * Called when the orientation of the device changes
+   * @private
+   */
+  orientationChanged_() {
+
+    const vidElement = this.video.element.querySelector('iframe, video');
+
+    // Determine which implementation of the orientation angle to use
+    const angle = this.ampdoc_.win.orientation !== undefined ?
+                  this.ampdoc_.win.orientation : screen.orientation.angle;
+
+    if (this.isVisible_
+    && this.getPlayingState() == PlayingStates.PLAYING_MANUAL
+    && (Math.abs(angle) == 270 || Math.abs(angle) == 90)) {
```

Comment:
what happens if user manually goes to fullscreen and then rotates in the following cases?
1- Chrome amp-video
2- Chrome amp-youtube
3- iOS amp-video
4- iOS amp-youtube

I am assuming (hoping) fullscreen request will be denied.